packages/api/src/entity/library_item.ts

```diff
@@ -138,9 +138,9 @@ export class LibraryItem {
   @Column('text')
   readableContent!: string
 
-  // NOT SUPPORTED IN TYPEORM
-  // @Column('vector', { nullable: true })
-  // embedding?: number[]
+  // typeorm does not support vector type, so we store it as a string
+  @Column('text')
+  embedding?: string
```

Comment:
Do you think we should use a separate table for these? I know we talked about it with some other attributes.